packages/debugger/src/context.ts

```diff
@@ -130,3 +131,52 @@ export function createTestContext<Code extends LocaleCode>(config: {
 
   return { scriptConfigs, executor };
 }
+
+export function getDefaultConfig(): {
+  deps: LocaleCode;
+} {
+  return {
+    deps: {
+      ALWAYS_SUCCESS: {
+        dep_type: "code",
+        path: path.join(__dirname, "../bin/always_success"),
+      },
+      ALWAYS_FAILURE: {
+        dep_type: "code",
+        path: path.join(__dirname, "../bin/always_failure"),
+      },
+      ANYONE_CAN_PAY: {
+        dep_type: "code",
+        path: path.join(__dirname, "../bin/anyone_can_pay"),
+      },
+      SUDT: {
+        dep_type: "code",
+        path: path.join(__dirname, "../bin/sudt"),
+      },
+      OMNI_LOCK: {
```

Comment:
```suggestion
      OMNILOCK: {
```
the Omnilock is named “Omnilock” in RFC, so we can just use `OMNILOCK` here